joss/paper.md

```diff
@@ -0,0 +1,168 @@
+   index: 2
+ - name: International Max Planck Research School on Earth System Modelling, Hamburg, Germany
+   index: 3
+date: 16 October 2020
+bibliography: paper.bib
+---
+
+<!-- 982 words total -->
+
+# Summary
+<!-- 71 words -->
+Predicting extreme events and variations in weather and climate provides crucial
+information for economic, social, and environmental decision-making [@Merryfield:2020]. However,
+quantifying prediction skill for multi-dimensional geospatial model output is
+computationally expensive and a difficult coding challenge. The large datasets
+(order gigabytes to terabytes) require parallel and out-of-memory computing to be
+analyzed efficiently. Further, aligning the many forecast initializations with differing
+observational products is a straight-forward, but exhausting and error-prone exercise
+for researchers.
+
+<!-- 68 words -->
+To simplify and standardize forecast verification across scales from hourly weather to
+decadal climate forecasts, we built `climpred`: a community-driven python package for
+computationally efficient and methodologically consistent verification of ensemble
+prediction models. The code base is maintained through open-source development. It
+leverages `xarray` [@Hoyer:2017] to anticipate core prediction ensemble dimensions
+(ensemble `member`, `init`ialization date and `lead` time) and `dask` [@dask] to
+perform out-of-memory and parallelized computations on large datasets.
+
+ <!-- 54 words -->
+`climpred` aims to offer a comprehensive set of analysis tools for assessing the quality
+of dynamical forecasts relative to verification products (_e.g._, observations,
+reanalysis products, control simulations). The package includes a suite of deterministic
+and probabilistic verification metrics that are constantly expanded by the community and
+are generally organized in our companion package, `xskillscore`.
+
+<!-- 161 words -->
+# Statement of Need
+While other climate verification packages exist (_e.g._, `s2dverification`
+[@Manubens:2018] written in R and `MurCSS` [@Illing:2014] written with python-based
+`CDO`-bindings [@CDO]), `climpred` is unique for many reasons. (1) It spans broad
+temporal scales of prediction, supporting the weather, subseasonal-to-seasonal (S2S),
+and seasonal-to-decadal (S2D) communities. (2) `climpred` supports `dask` [@dask] and
+thus works across all computational scales, from personal laptops to supercomputers
+(HPC), and in the cloud. This leads to, for example, verification of a global 5°x5°
+resolution climate prediction in less than one second, compared to the 8 minutes
+required in `MurCSS`.  (3) `climpred` is highly modular and supports the research
+process from end-to-end, from loading in model output, to interactive pre-processing and
+analysis, to visualization. (4) `climpred` is part of and benefits from the wider
+scientific python community, `pangeo` [@Eynard:2019]. A wide adoption of `climpred`
```

Comment:
is this the best pangeo reference there is?

Reply:
I only know of this one as an alternative: 
Odaka, T. E., Banihirwe, A., Eynard-Bontemps, G., Ponte, A., Maze, G., Paul, K., Baker, J., & Abernathey, R. (2020). The Pangeo Ecosystem: Interactive Computing Tools for the Geosciences: Benchmarking on HPC. G. Juckeland & S. Chandrasekaran (Eds.), Tools and Techniques for High Performance Computing (pp. 190–204). Cham: Springer International Publishing. doi: 10/ggqs7s

Reply:
I spoke with Ryan on gitter and that's what he suggested. Said they're still working on a formal paper.